CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ CameraOptions(
     )))
 ```
 
+* Bump Pigeon to 17.1.2
+* [iOS] Fix crash in `onStyleImageMissingListener`.
+* Fix camera center not applied from map init options.
+
```

Comment:
these lines are duplicated, they were moved to the top of the changelog in previous commit already